### PR TITLE
Feature/upgrade phoenix 1 2 1

### PR DESCRIPTION
**Pull request by hecbuma**

### CODE REVIEW

File: web/models/post.ex [hunk 1]
```diff
@@ -13,29 +13,30 @@ defmodule Dash.Post do
...
 
-    timestamps
+    timestamps()
   end
 
   @required_fields ~w(title)
   @optional_fields ~w(body permalink tags summary published published_at user_id)
 
-  after_load :flat_tags
+  # after_load :flat_tags
```
Comment (mariochavez) [inline web/models/post.ex]: just remove this line

File: config/dev.exs [hunk 2]
```diff
@@ -10,33 +10,32 @@ config :dash, Dash.Endpoint,
...
+# in production as building large stacktraces may be expensive.
 config :phoenix, :stacktrace_depth, 20
 
 # Configure your database
 config :dash, Dash.Repo,
   adapter: Ecto.Adapters.Postgres,
   database: "dash_dev",
+  hostname: "localhost",
   pool_size: 10
-
```
Comment (mariochavez) [inline config/dev.exs:40]: how would you access the admin area from development?

**Comment by mariochavez** [comment]
Do we need Phoenix logo?

**Comment by EduardoBautista** [comment]
@mariochavez Yes, otherwise how will people know that this is webscale?

**Pull request merged**